components/inputs/input-label-styles.js

```diff
@@ -3,14 +3,13 @@ import { css } from 'lit-element/lit-element.js';
 export const inputLabelStyles = css`
 	.d2l-input-label {
 		cursor: default;
-		display: block;
+		display: inline-block;
 		font-size: 0.7rem;
 		font-weight: 700;
 		letter-spacing: 0.2px;
 		line-height: 1rem;
-		margin: 0;
-		padding: 0 0 7px 0;
-		width: 100%;
+		margin: 0 0 7px 0;
+		padding: 0;
```

Comment:
In order for label skeletons to not be full-width, I switched them to `inline-block`. I moved the padding to margin so that the `7px` at the bottom would be outside the skeleton.